test_runner/src/test/kotlin/ftl/args/AndroidArgsTest.kt

```diff
@@ -1252,6 +1246,32 @@ AndroidArgs
         val parsedYml = AndroidArgs.load(yaml)
         runBlocking { runAndroidTests(parsedYml) }
     }
+
+    /**
+     * Double directory check for regressive issue protect, first to check directory
+     * really not exist on disk (when issue come back and directory exists, assert not throws),
+     * second for checking directory not created after load args
+     * it's should be enough
+     */
+    @Test
+    fun `should not throw if directory not exists`() {
+        val resultsDir = UUID.randomUUID().toString()
+        val directoryPath = Paths.get(resultsDir)
+        if (Files.exists(directoryPath)) {
+            Assert.fail("Test directory ($resultsDir) shouldn't exists! It's remote directory.")
+        }
+        val yaml = """
+        gcloud:
+          app: $appApk
+          test: $testApk
+          results-dir: $resultsDir
+        """.trimIndent()
+        AndroidArgs.load(yaml)
+        if (Files.exists(directoryPath)) {
+            Assert.fail("Test directory ($resultsDir) shouldn't be created! It's remote directory on cloud!")
+        }
+    }
 }
 
-private fun AndroidArgs.Companion.load(yamlData: String, cli: AndroidRunCommand? = null): AndroidArgs = load(StringReader(yamlData), cli)
+private fun AndroidArgs.Companion.load(yamlData: String, cli: AndroidRunCommand? = null): AndroidArgs =
+    load(StringReader(yamlData), cli)
```

Comment:
format change? 🙂

Reply:
Sadly yes. I should check all files modified in this pr not only one. I revert changes. In future I will pay more attention for that.